app/assets/stylesheets/thredded/components/_post.scss

```diff
@@ -4,6 +4,71 @@
   @include thredded-media-mobile {
     margin-bottom: $thredded-base-spacing;
   }
+  .thredded--post--dropbtn .thredded--post--three-dot-menu {
```

Comment:
1. Can we remove the `.thredded--post--dropbtn` wrapper, as it doesn't seem necessary?
2. Let's rename `.thredded--post--three-dot-menu` to `.thredded--post--dropdown--toggle`.